audiobook/main.py

```diff
@@ -99,9 +97,10 @@ def save_audio(self, input_book_path, password=None, save_page_wise=False, extra
         json_book, metadata = self.create_json_book(input_book_path, password, extraction_engine)
 
         book_name = metadata["book_name"]
-        os.makedirs(book_name, exist_ok=True)
+        book_dir = os.path.join(BOOK_DIR, book_name)
+        os.makedirs(book_dir, exist_ok=True)
 
-        print("Saving audio files in folder: {}".format(book_name))
+        print("Saving audio files in folder: {}".format(book_dir))
```

Comment:
Saving audio to the same directory that contains books makes more sense to me than the actual dir where the code lives